### PR TITLE
Request to merge Create table

### PR DESCRIPTION
**Pull request by Kingstalux**
- In this project, I used a relational database to create the initial data structure for a vet clinic. 
- I created a table to store animals' information, insert some data into it, and query it.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals
+WHERE name LIKE '%mon%';
+
+SELECT name FROM animals
+WHERE date_of_birth > '2016-01-01' and date_of_birth < '2019-01-01';
+
```
Comment (ElijahTobs) [inline queries.sql]: - [x] This should be:

```ruby
WHERE date_of_birth >= '2016-01-01' and date_of_birth <= '2019-01-01';

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @Kingstalux  👋

You've done really well so far! 👍 

## Major Highlights 💯 

Your README is professional ✔️ 
You've got good commit messages ✔️ 
You have a good understanding of what is required of you✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: queries.sql (hunk 1 above)
Comment (Kingstalux) [reply queries.sql]: okay, I'll change that.

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @Kingstalux , 

Good job making the changes required.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**